server/search/searchDB.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
     //this is scuffed
     //because cache is not set up yet to based on auth.  A sample insert is made
     //after sample is set ---- comment out
-    // let sampleKey = 'Iron Man 2'
-    // let sampleValue = '10138'
-    // let sample = redis.set(sampleKey, sampleValue)
+    let sampleKey = 'Iron Man 2'
```

Comment:
By Robert's design, this should not be left uncommented in the working app. It needs to run only once in order to set up test data.